portfolio.js

```diff
@@ -267,4 +267,36 @@ form.addEventListener('submit', (e) => {
     formButton.reportValidity();
     e.preventDefault();
   }
-});
\ No newline at end of file
+});
+
+// set up local storage
+
+const contactForm = document.getElementById('contact-form');
+const nameInput = document.getElementById('name');
+const mailInput = document.getElementById('email');
+
+contactForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  // store user inputs in an object
+
+  const storageObject = {
```

Comment:
- [ ] Nice working on the form data object, kindly add all the input options like the previous reviewer said, you did not add `message` input.

 ```
const storageObject = {
    name: nameInput.value,
    email: mailInput.value,
    message: messageInput.value,
  }
```